src/RcppExports.cpp

```diff
@@ -6,6 +6,11 @@
 
 using namespace Rcpp;
 
+#ifdef RCPP_USE_GLOBAL_ROSTREAM
```

Comment:
Not shure why `devtools::build()` added these lines.

Reply:
New Rcpp does that, i.e. it is in Rcpp 1.0.7 but wasn't in Rcpp last time the file was updated (which could have been a while).